src/main/typescript/actions/authenticationActions.ts

```diff
@@ -63,45 +57,7 @@ export const cookieAuthenticate: () => ComplexThunkAction = () => async (dispatc
      */
     dispatch(authenticatePending)
 
-    try {
-        await fetch.post(loginUrl(getState()))
-
-        dispatch(fetchUserOnLogin())
-    }
-    catch (loginResponse) {
-        LocalStorage.setLogout()
-
-        dispatch(cookieAuthenticateRejected)
-    }
-}
-
-export const authenticate: (userName: string, password: string) => ComplexThunkAction = (userName, password) => async (dispatch, getState) => {
-    /*
-     * dispatch AUTH_LOGIN_PENDING
-     * call server with 'auth/login'
-     *   - success:
-     *       dispatch fetchUserOnLogin
-     *   - failure:
-     *       local storage --> remove 'logged-in'
-     *       dispatch AUTH_LOGIN_REJECTED
-     */
-    dispatch(authenticatePending)
-
-    try {
-        await fetch.post(loginUrl(getState()), {}, {
-            auth: {
-                password: password,
-                username: userName,
-            },
-        })
-
-        dispatch(fetchUserOnLogin())
-    }
-    catch (loginResponse) {
-        LocalStorage.setLogout()
-
-        dispatch(authenticateRejected(`Not able to login: ${loginResponse.response.data}`))
-    }
+    dispatch(fetchUserOnLogin())
```

Comment:
since `fetchUserOnLogin` is now only called once, please merge these two functions. That way you also don't need the extra `dispatch` here.

also merge/update the comments that provide all the steps taken